utils/libafl_benches/benches/rand_speeds.rs

```diff
@@ -2,16 +2,19 @@
 
 use criterion::{black_box, criterion_group, criterion_main, Criterion};
 use libafl_bolts::rands::{
-    Lehmer64Rand, Rand, RomuDuoJrRand, RomuTrioRand, XorShift64Rand, Xoshiro256StarRand,
+    Lehmer64Rand, Rand, RomuDuoJrRand, RomuTrioRand, Sfc64Rand, XorShift64Rand,
+    Xoshiro256PlusPlusRand,
 };
 
 fn criterion_benchmark(c: &mut Criterion) {
+    let mut sfc64 = Sfc64Rand::with_seed(1);
     let mut xorshift = XorShift64Rand::with_seed(1);
-    let mut xoshiro = Xoshiro256StarRand::with_seed(1);
+    let mut xoshiro = Xoshiro256PlusPlusRand::with_seed(1);
     let mut romu = RomuDuoJrRand::with_seed(1);
     let mut lehmer = Lehmer64Rand::with_seed(1);
     let mut romu_trio = RomuTrioRand::with_seed(1);
 
+    c.bench_function("sfc64", |b| b.iter(|| black_box(sfc64.next())));
```

Comment:
Random question, how does this guy do in the benchmark?

Reply:
Sub-nanosecond `u64` generation on modern machines (I am fighting with turbo boost on my laptop, so can't get stable numbers to report right now). Slower than `Romu` (which in general is considered risky due to having no guarantees regarding bias or cycle length), but faster than `xoshiro256++` (which is among the fastest widely-used general-purpose PRNGs today).

I'd say this is as fast as you can go without sacrificing something (like `Romu` and `Lehmer64` do).